app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ class UsersController < ApplicationController
   rescue_from ActionController::InvalidAuthenticityToken, with: :login_token_reload
   skip_before_action :require_mail, :only => [:edit, :update, :logout, :stop_impersonation]
   skip_before_action :require_login, :check_user_enabled, :authorize, :session_expiry, :update_activity_time, :set_taxonomy, :set_gettext_locale_db, :only => [:login, :logout, :extlogout]
-  skip_before_action :authorize, :only => [:extlogin, :impersonate, :stop_impersonation]
+  skip_before_action :authorize, :only => [:extlogin, :impersonate, :stop_impersonation, :invalidate_jwt]
```

Comment:
The same as for the API endpoint. You can specify the new action in the security block:
https://github.com/theforeman/foreman/blob/eb66c09e63ba9b04f3d86c240ec25de52647bc84/config/initializers/f_foreman_permissions.rb#L565